tests/test_gaia_direct.py

```diff
@@ -0,0 +1,107 @@
+import os
+from pathlib import Path
+import unittest
+import numpy as np
+import pandas as pd
+import lsst.daf.butler as daf_butler
+from skycatalogs import skyCatalogs
+from skycatalogs.objects import GaiaCollection
+
+
+PACKAGE_DIR = os.path.dirname(os.path.abspath(str(Path(__file__).parent)))
+SKYCATALOG_ROOT = os.path.join(PACKAGE_DIR, "skycatalogs", "data")
+CATALOG_DIR = os.path.join(PACKAGE_DIR, "skycatalogs", "data", "ci_sample")
+
+CONFIG = {'area_partition': None,
+          'butler_parameters':
+          {'collections': 'refcats',
+           'dstype': 'gaia_dr2_20200414',
+           'repo': os.path.join(CATALOG_DIR, 'repo')},
+          'data_file_type': 'butler_refcat'}
```

Comment:
It's a bit confusing to see `data_file_type` set as `'butler_refcat'` here when this test code is mean for direct FITS access.  Unlike `test_gaia_objects.py`, `data_file_type` doesn't need to be set since this version only reads in the data directly into `self.df`, so I think it would be clearer to omit this and maybe note that this `CONFIG` dict is just used by `get_gaia_data`.